src/pages/_app.tsx

```diff
@@ -1,13 +1,75 @@
-import { type AppType } from 'next/app'
-import { type Session } from 'next-auth'
+// src/pages/_app.tsx
+import { httpBatchLink } from '@trpc/client/links/httpBatchLink'
+import { loggerLink } from '@trpc/client/links/loggerLink'
+import { withTRPC } from '@trpc/next'
 import { SessionProvider } from 'next-auth/react'
-import { ChakraProvider } from '@chakra-ui/react'
-import { theme } from '~/config/ChakraTheme'
-import { trpc } from '../utils/trpc'
+import superjson from 'superjson'
+import type { AppType } from 'next/app'
+import type { AppRouter } from '~/server/trpc/router/_app'
+import type { Session } from 'next-auth'
+import { ChakraProvider, extendTheme, type ThemeConfig } from '@chakra-ui/react'
 import { Provider } from 'react-redux'
 import store from '~/store/store'
 import '../styles/globals.css'
 
+export const theme: ThemeConfig = extendTheme(
```

Comment:
The existing configuration can be found in `src/config/ChakraTheme.ts`. Just copy and paste the relevant code over to the file and delete this portion.